changedetectionio/fetch_site_status.py

```diff
@@ -1,4 +1,6 @@
 import hashlib
+import urllib3
```

Comment:
why is this here?

Reply:
must have forgotten to remove it, must have duplicated this when was resolving conflicts from master. I removed all unused imports now